src/index.html

```diff
@@ -0,0 +1,376 @@
+                <option value=4>4</option>
+                <option value=5>5</option>
+                <option value=6>6</option>
+                <option value=7>7</option>
+                <option value=8>8</option>
+                <option value=9>9</option>
+                <option value=10>10</option>
+              </select>
+            </td>
+            <td>
+              <select id="roll-10.1">
+                <option value=""></option>
+                <option value=1>1</option>
+                <option value=2>2</option>
+                <option value=3>3</option>
+                <option value=4>4</option>
+                <option value=5>5</option>
+                <option value=6>6</option>
+                <option value=7>7</option>
+                <option value=8>8</option>
+                <option value=9>9</option>
+                <option value=10>10</option>
+              </select>
+            </td>
+            <td>
+              <select id="roll-10.2">
+                <option value=""></option>
+                <option value=1>1</option>
+                <option value=2>2</option>
+                <option value=3>3</option>
+                <option value=4>4</option>
+                <option value=5>5</option>
+                <option value=6>6</option>
+                <option value=7>7</option>
+                <option value=8>8</option>
+                <option value=9>9</option>
+                <option value=10>10</option>
+              </select>
+            </td>
+          </tr>
+          <tr id="scores">
+            <td colspan="2" id="round-1"></td>
+            <td colspan="2" id="round-2"></td>
+            <td colspan="2" id="round-3"></td>
+            <td colspan="2" id="round-4"></td>
+            <td colspan="2" id="round-5"></td>
+            <td colspan="2" id="round-6"></td>
+            <td colspan="2" id="round-7"></td>
+            <td colspan="2" id="round-8"></td>
+            <td colspan="2" id="round-9"></td>
+            <td colspan="2" id="round-10"></td>
+          </tr>
+          <tr id="cumulative">
+            <td colspan="2" id="cumulative-1"></td>
+            <td colspan="2" id="cumulative-2"></td>
+            <td colspan="2" id="cumulative-3"></td>
+            <td colspan="2" id="cumulative-4"></td>
+            <td colspan="2" id="cumulative-5"></td>
+            <td colspan="2" id="cumulative-6"></td>
+            <td colspan="2" id="cumulative-7"></td>
+            <td colspan="2" id="cumulative-8"></td>
+            <td colspan="2" id="cumulative-9"></td>
+            <td colspan="2" id="cumulative-10"></td>
+          </tr>
+        </table>
+      </div>
+
+
+      <footer id="footer" class="footer py-2 px-3">
+      </footer>
+    </div>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js"></script>
```

Comment:
Every week I ask Rick this question and every week he refuses to answer! Maybe you can: why does the application need `tether`?